### PR TITLE
Pr/308

### PR DESCRIPTION
**Pull request by Reaver01**
Short Description: Finally no conflicts

Fixes:
- Map
- 
-

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -17,237 +17,91 @@
...
           setInterval(updateTrainer, 1000);
+          setInterval(addCatchable, 1000);
         }, 5000);
       }, 5000);
       }
       function placeTrainer() {
-        loadJSON('location.json?_=' + new Date().getTime(),
-          function(data) { loc = data; map.setZoom(16); map.panTo({lat: parseFloat(loc.lat), lng: parseFloat(loc.lng)}); },
+        loadJSON('location.json',
+          function(data) { tLoc = data; map.setZoom(16); map.panTo({lat: parseFloat(tLoc.lat), lng: parseFloat(tLoc.lng)}); },
```
Comment (solderzzc) [inline web/index.html]: Then will not move on the chrome, browser use cache.

Comment (Reaver01) [inline web/index.html]: That line makes my browser load nothing

Comment (solderzzc) [inline web/index.html]: oh.... which browser you are using?

Comment (Reaver01) [inline web/index.html]: Firefox Web Developer

Comment (solderzzc) [inline web/index.html]: That's not issue on browser imo, I am using the nodejs http-server. Let me test python server.

Comment (Reaver01) [inline web/index.html]: oh.  I am not serving it. that may be why

Comment (solderzzc) [inline web/index.html]: Can't wait for the merge.

Comment (Reaver01) [inline web/index.html]: I am going to bed.  You guys can figure out whether you want to merge this I guess.

Comment (solderzzc) [inline web/index.html]: Night, works :)

**Pull request merged**